xml/gl.xml

```diff
@@ -47752,7 +47752,7 @@ typedef unsigned int GLhandleARB;
                 <command name="glGetFramebufferParameterivMESA"/>
             </require>
         </extension>
-        <extension name="GL_MESA_framebuffer_swap_xy" supported="gles2">
+        <extension name="GL_MESA_framebuffer_swap_xy" supported="gl|gles2">
```

Comment:
If this extension is also supported in core profile, you have to manually specify it.

```suggestion
        <extension name="GL_MESA_framebuffer_swap_xy" supported="gl|glcore|gles2">
```

Reply:
added